examples/dynamic-client/namespaced_custom_resource.py

```diff
@@ -21,10 +21,44 @@
 """
 
 from kubernetes import config, dynamic
+from kubernetes import client as k8s_client
 from kubernetes.dynamic.exceptions import ResourceNotFoundError
 from kubernetes.client import api_client
 import time
 
+def list_ingressroute_for_all_namespaces(group, version, plural):
+    custom_object_api = k8s_client.CustomObjectsApi()
+
+    list_of_ingress_routes = custom_object_api.list_cluster_custom_object(
+        group, version, plural
+    )
+    print(
+        "%s\t\t\t%s\t\t\t%s\t\t%s\t\t\t\t%s"
+        % ("NAME", "NAMESPACE", "FQDN", "TLS", "STRATEGY")
+    )
+    for item in list_of_ingress_routes["items"]:
+        print(
+            "%s\t%s\t\t%s\t%s\t%s"
+            % (
+                item["metadata"]["name"],
+                item["metadata"]["namespace"],
+                item["spec"]["virtualhost"]["fqdn"],
+                item["spec"]["virtualhost"]["tls"],
+                item["spec"]["strategy"]
+            )
+        )
+
+def create_namespace(namespace_api, name):
+    namespace_manifest = {
+        "apiVersion": "v1",
+        "kind": "Namespace",
+        "metadata": {"name": name, "resourceversion": "v1"},
+    }
+    namespace = namespace_api.create(body=namespace_manifest)
+
+
+def delete_namespace(namespace_api, name):
+    namespace_deleted = namespace_api.delete(name=name)
```

Comment:
namespace_deleted is not used, do you need to assign to it?

Reply:
Thank you for pointing. No, the variable `namespace_deleted` is not required.

I've removed it.

Reply:
Thank you for pointing. No, the variable `namespace` is not required. I've removed it.